yt_dlp/extractor/adobetv.py

```diff
@@ -84,22 +84,23 @@ def _parse_video_data(self, video_data):
 
 
 class AdobeTVEmbedIE(AdobeTVBaseIE):
+    _WORKING = False
     IE_NAME = 'adobetv:embed'
     _VALID_URL = r'https?://tv\.adobe\.com/embed/\d+/(?P<id>\d+)'
-    _TEST = {
+    _TESTS = [{
         'url': 'https://tv.adobe.com/embed/22/4153',
         'md5': 'c8c0461bf04d54574fc2b4d07ac6783a',
         'info_dict': {
             'id': '4153',
             'ext': 'flv',
             'title': 'Creating Graphics Optimized for BlackBerry',
             'description': 'md5:eac6e8dced38bdaae51cd94447927459',
-            'thumbnail': r're:https?://.*\.jpg$',
+            'thumbnail': r're:https?://.+\.jpg',
```

Comment:
The `$` matches the end of the filename. Is it expected that the value will end in `\.jpg\w`, or that query/fragment parts may be present (`$` -> `(?:$|[#?])`)?

Ofc, moot if `not _WORKING`, but this may apply to similar proposed changes.